packages/mobile/test/schemas.ts

```diff
@@ -947,7 +947,6 @@ export const v32Schema = {
   account: {
     ...v31Schema.account,
     hasLinkedBankAccount: false,
-    shouldSkipOnboardingEducationScreen: undefined,
```

Comment:
probly doesn't matter since it looks like this was added by accident in the original PR (not seeing any redux state that made it into the final version of the PR). but in general this is the kind of thing to avoid reverting, since it's a schema migration-- instead, we'd need a new migration to drop the thing we're getting rid of